lib/commands/subscribe.js

```diff
@@ -2,6 +2,8 @@ const {
   Subscribed, NotFound, AlreadySubscribed, NotSubscribed, UpdatedSettings,
 } = require('../messages/flow');
 
+const isSlackbot = require('../is-slackbot');
```

Comment:
nit: `isSlackbot` might confuse someone who's looking at this code in the future. Could we maybe call this something like `originatesFromSlack` or `isRequestFromSlack`?